library/core/src/ops/control_flow.rs

```diff
@@ -177,9 +178,12 @@ impl<B, C> ControlFlow<B, C> {
     /// ```
     #[inline]
     #[unstable(feature = "control_flow_enum", reason = "new API", issue = "75744")]
-    pub fn break_value(self) -> Option<B> {
+    pub const fn break_value(self) -> Option<B>
+    where
+        C: ~const Drop,
+    {
         match self {
-            ControlFlow::Continue(..) => None,
+            ControlFlow::Continue(_x) => None,
```

Comment:
what's up with that? please add a `HACK` comment referencing the issue that will allow us to go back to the previous

Reply:
See https://rust-lang.zulipchat.com/#narrow/stream/213817-t-lang/topic/Exact.20semantics.20for.20match.20and.20drop.20glue.20emitted